samples/net/civetweb/websocket_server fails to build

**Describe the bug**

Trying to build samples/net/civetweb/websocket_server fails

**To Reproduce**
Steps to reproduce the behavior:
1. mkdir samples/net/civetweb/websocket_server/build; cd samples/net/civetweb/websocket_server/build
2. cmake -DBOARD=nucleo_h745zi_q_m7
3. make
4. See error:

```
[  3%] Generating zephyr/include/generated/web_page/ws.js.inc
/bin/sh: /home/galak/git/zephyr/samples/net/civetweb/websocket_server/build/zephyr/include/generated//web_page/ws.js.inc: No such file or directory
make[2]: *** [CMakeFiles/gen_ws_js_inc_9dd2249e742a278f4e7dceadee74a807.dir/build.make:80: zephyr/include/generated/web_page/ws.js.inc] Error 1
make[1]: *** [CMakeFiles/Makefile2:2407: CMakeFiles/gen_ws_js_inc_9dd2249e742a278f4e7dceadee74a807.dir/all] Error 2
make: *** [Makefile:103: all] Error 2
```

This is due to the fact that the dir `build/zephyr/include/generated//web_page/` doesn't get created.

--- a/samples/net/civetweb/websocket_server/CMakeLists.txt
+++ b/samples/net/civetweb/websocket_server/CMakeLists.txt
@@ -30,6 +30,8 @@ target_sources(app PRIVATE
 set(gen_dir ${ZEPHYR_BINARY_DIR}/include/generated/)
 set(web_page_dir web_page)
 
+file(MAKE_DIRECTORY ${gen_dir}/${web_page_dir})
+
 # List of files that are used to generate .h file that can be included
 # into .c file.
 foreach(inc_file

